cpp/server/ct-server.cc

```diff
@@ -34,6 +35,7 @@ DEFINE_string(trusted_cert_file, "",
               "File for trusted CA certificates, in concatenated PEM format");
 DEFINE_string(cert_dir, "", "Storage directory for certificates");
 DEFINE_string(tree_dir, "", "Storage directory for trees");
+DEFINE_string(meta_dir, "", "Storage directory for meta info");
```

Comment:
Maybe not in this pull request, but I've always found it a bit odd to have to specify each directory separately (instead of just having one, and ```ct-server``` figuring out how it wants to use it), and this is making it even more clear...